lightkurve/periodogram.py

```diff
@@ -74,4 +79,272 @@ def plot(self, frequency=None, scale="linear", ax=None, numax=None, **kwargs):
+
+        find_numax() method first estimates the background trend
+        and then smoothes the power spectrum using a gaussian filter.
+        Peaks are then determined in the smoothed power spectrum.
+
+        Returns:
+        --------
+        nu_max : float
+            The nu max of self.powers. Nu max is in microhertz
+        """
+
+        bkg = self.estimate_background(self.frequencies.value, self.powers.value)
+        df = self.frequencies[1].value - self.frequencies[0].value
+        smoothed_ps = gaussian_filter(self.powers.value / bkg, 10 / df)
+        peak_freqs = self.frequencies[self.find_peaks(smoothed_ps)].value
+        nu_max = peak_freqs[peak_freqs > 5][0]
+        return nu_max
+
+    def estimate_delta_nu(self, numax=None):
+        """Estimates delta nu value, or large frequency spacing
+
+        Estimates the delta nu value centered around the numax value given. The autocorrelation
+        function will find the distancing between consecutive modesself.
+
+        Parameters
+        ----------
+        numax : float
+            The Nu max value to center our autocorrelation function
+
+        Returns
+        -------
+        delta_nu : float
+            So-called large frequency spacing
+        """
+        def next_pow_two(n):
+            i = 1
+            while i < n:
+                i = i << 1
+            return i
+
+        def acor_function(x):
+            x = np.atleast_1d(x)
+            n = next_pow_two(len(x))
+            f = np.fft.fft(x - np.mean(x), n=2*n)
+            acf = np.fft.ifft(f * np.conjugate(f))[:len(x)].real
+            acf /= acf[0]
+            return acf
+
+        # And the autocorrelation function of a lightly smoothed power spectrum
+        bkg = self.estimate_background(self.frequencies.value, self.powers.value)
+        df = self.frequencies[1].value - self.frequencies[0].value
+        acor = acor_function(gaussian_filter(self.powers.value / bkg, 0.5 / df))
+        lags = df*np.arange(len(acor))
+        acor = acor[lags < 30]
+        lags = lags[lags < 30]
+
+        if numax is None:
+            raise ValueError("Must provide a nu max value")
+        # Expected delta_nu: Stello et al (2009)
+        dnu_expected = 0.263 * numax ** 0.772
+        peak_lags = lags[self.find_peaks(acor)]
+        delta_nu = peak_lags[np.argmin(np.abs(peak_lags - dnu_expected))]
+        return delta_nu
+
+    def estimate_background(self, x, y, log_width=0.01):
+        """Estimates background noise
+
+        Estimates the background noise via median value
+
+        Parameters
+        ----------
+        self : Periodogram object
+            Periodogram object
+        x : array-like
+            Time measurements
+        y : array-like
+            Flux measurements
+        log_width : float
+            The error range
+
+        Returns
+        -------
+        bkg : array-like
+            background trend
```

Comment:
Does this have a unit?  If so, let's mention it.

Reply:
Nope! This is unitless.